mavros_msgs/State    mav_state
bool offboard
uint8 tol_state
